src/containerbuddy/config.go

```diff
@@ -137,7 +138,7 @@ func parseConfig(configFlag string) (*Config, error) {
 }
 
 // determine the IP address of the container
-func getIp(usePublic bool) string {
+func getIp(usePublic bool, interfaceName string) string {
```

Comment:
If we get rid of the no-longer-needed `usePublic` arg, we can simplify this function a lot and eliminate the `isPublicIp` function below.